workflow/common/convert.go

```diff
@@ -26,16 +26,21 @@ func ConvertCronWorkflowToWorkflow(cronWf *wfv1.CronWorkflow) *wfv1.Workflow {
 	return wf
 }
 
-func ConvertWorkflowTemplateToWorkflow(template *wfv1.WorkflowTemplate) *wfv1.Workflow {
-	wf := toWorkflow(template.TypeMeta, template.ObjectMeta, template.Spec.WorkflowSpec)
-	wf.Labels[LabelKeyWorkflowTemplate] = template.ObjectMeta.Name
-	return wf
-}
-
-func ConvertClusterWorkflowTemplateToWorkflow(template *wfv1.ClusterWorkflowTemplate) *wfv1.Workflow {
-	wf := toWorkflow(template.TypeMeta, template.ObjectMeta, template.Spec.WorkflowSpec)
-	wf.Labels[LabelKeyClusterWorkflowTemplate] = template.ObjectMeta.Name
+func ConvertWorkflowTemplateToWorkflow(templateName string, clusterScope bool) *wfv1.Workflow {
```

Comment:
This function should be renamed since "convert" is misleading. I suggest `NewWorkflowFromWorkflowTemplate`